app/src/main/res/values/dimens.xml

```diff
@@ -12,6 +12,6 @@
   <dimen name="weather_icon_width">150dp</dimen>
   <dimen name="weather_icon_height">150dp</dimen>
 
-  <dimen name="face_recognition_text">100sp</dimen>
+  <dimen name="face_recognition_text">50sp</dimen>
```

Comment:
I reduced the size because I was having some scaling problem in my emulator. The emojis-TextViews are debug only so it's fine. Right?